app/controllers/maintaining/site_updates_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Maintaining::SiteUpdatesController < ApplicationController
+  def index
+    @updates = SiteUpdate.sorted.page(params[:page]).per(20)
+  end
+
+  def edit
+    @update = SiteUpdate.find(params[:id])
+  end
+
+  def update
+    @update = SiteUpdate.find(params[:id])
```

Comment:
Can `find` return `nil`? (also applies to the `edit` method)

Reply:
No, find raises an `ActiveRecord::NotFound` exception if it can't find the record, which causes the page to 404.

Reply:
Okay cool